base/system/autochk/autochk.c

```diff
@@ -5,7 +5,7 @@
  * PURPOSE:         Filesystem checker
  * PROGRAMMERS:     Aleksey Bragin
  *                  Eric Kohl
- *                  Herv� Poussineau
+ *                  Hervé Poussineau
```

Comment:
Not sure why this registered as a change.

Reply:
Possibly because there is a newline character change here.